src/components/Admin/SelectUser/index.tsx

```diff
@@ -74,7 +74,7 @@ const SelectUser = observer((props: Props) => {
     if (!currentPage) {
         return null;
     }
-    const users = currentPage.activeStudentGroup?.students || userStore.users;
+    const users = currentPage.activeStudentGroup?.students || userStore.managedUsers;
```

Comment:
`managedUsers` are all users that are managed/administrated by the current user, either:
     * - when the current user is an admin, or
     * - through a group admin-membership

Reply:
Makes sense, and I think the name is appropriate.